### DISCUSSION TITLE
Unable to add borderRadius in tailwind.config.js. Inside theme.extend property

### DISCUSSION THREAD

**Discussion by asaurabhprsas** [tailwindlabs/tailwindcss#13776] [Help]
```js
// It works fine
{
  theme: {
    borderRadius: {
      DEFAULT: '10px'
    }
  }
}

// Its not possible
{
  theme: {
    extend: {
      borderRadius: {
        newProperty: "20px"
      }
    }
  }
}
```

tailwind version = "tailwindcss": "^3.4.3"

**Comment by wongjn** [comment]
Seems to work [in this Tailwind Play](https://play.tailwindcss.com/Sv1jrQ7QTv) which would suggest it could be some other misconfiguration or misuse in your project that is causing your unexpected behavior. Please consider providing a project that reproduces the undesired behavior.

**Reply by asaurabhprsas** [reply]
Hey, Thank you for your time.
Hers's the link of repo.

[LINK](https://github.com/asaurabhprsas/html-sass-tailwind-boilerplate)

**Reply by wongjn** [reply]
Thank you. So it seems you haven't actually added your custom border radius to [the Tailwind configuration](https://github.com/asaurabhprsas/html-sass-tailwind-boilerplate/blob/ef5d5ea7dadb2ddab5b7778b89ef183fc1b8bc55/tailwind.config.js). Consider adding it to your `tailwind.config.js` file like:
```diff
 theme: {
-  extend: {},
+  extend: {
+    borderRadius: {
+      newProperty: "20px"
+    }
+  }
 },
```

Also, your build system seems to be converting:

```html
<link rel="stylesheet" href="assets/css/style.css">
```
to
```html
<link rel="assets/css/style.css">
```
Causing the CSS to not load properly.

Furthermore, it does not seem you've used the `round-newProperty` string in any of the code that is scanned by Tailwind.
